src/babylon.engine.ts

```diff
@@ -3362,26 +3359,32 @@
 
         // Loading screen
         public displayLoadingUI(): void {
+            if (!this._loadingScreen) this._loadingScreen = new DefaultLoadingScreen(this._renderingCanvas)

```

Comment:
what about creating a loadingScreen property? this way you could instantiate it on first get?